### ISSUE TITLE
@filter cors for DELETE

### ISSUE THREAD

**Issue by dmenne** [trestletech/plumber#409]
The filter mentioned in #143 works for GET requests, but is not active for DELETE. When I convert my DELETE to a (misnomed) GET, everything is fine.

```
#* @filter cors
cors <- function(res) {
  res$setHeader("Access-Control-Allow-Origin", "*")
  plumber::forward()
}
```

**Comment by dmenne** [comment]
@shizidushu workaround in #66 works for me. I leave this issue open nevertheless, because it might help others until an official method is documented.

**Comment by anv42** [comment]
Hello,

I'm facing issues with CORS requests from chrome to a Plumber API hosted on RSC. I've added the following filter as suggested in the documentation : 

#* @filter cors
cors <- function(req, res) {
  res$setHeader('Access-Control-Allow-Origin', '*')
  plumber::forward()
}

But, I'm still getting a 404 status. The API is otherwise working fine with a CURL statement. Has this issue been resolved in 0.4.6 or is there a workaround?

**Comment by schloerke** [comment]
@anv42 Did https://github.com/trestletech/plumber/issues/66#issuecomment-418660334 work for you?  If not, please open a new issue on plumber.  Thank you.

**Comment by anv42** [comment]
@schloerke No, [#66 (comment)](https://github.com/trestletech/plumber/issues/66#issuecomment-418660334) didn't work either. I'll open a new ticket with as much detail into the problem as possible.
